src/features/patients/pages/PatientForm.tsx

```diff
@@ -110,7 +151,16 @@ export const PatientForm: FC = () => {
           onChange={handlePatientChange}
           refresh={refreshForms}
         >
-          <div />
+          <Button
+            ref={buttonRef}
+            variant="contained"
+            color="primary"
+            type="submit"
+            onClick={() => "submit!"}
+            style={{ display: "none" }}
+          >
+            Submit
+          </Button>
```

Comment:
so this button triggers the event for this form, but is hidden.

Very coupled to having a survey at this point

Reply:
Yes, the tight coupling is a concern